internal/service/error.go

```diff
@@ -13,3 +16,21 @@ func (e Err) Error() string {
 func (e Err) Unwrap() error {
 	return e.Err
 }
+
+type InvalidInputErr struct {
+	Field string
+}
+
+func (e InvalidInputErr) Error() string {
+	return fmt.Sprintf("Invalid value for field: %s", e.Field)
+}
+
+type InvalidFilter struct {
+	Filter string
+}
+
+func (e InvalidFilter) Error() string {
+	return fmt.Sprintf("Invalid filter for search: %s", e.Filter)
+}
+
+var InvalidPassword = errors.New("Can't login, passwords doesn't match")
```

Comment:
This is a small fix. The block of variable definitions use to be at the top of the file after the library import section.